projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts

```diff
@@ -441,7 +441,15 @@ export class SpeciesSelectorComponent implements OnInit, OnDestroy {
         });
         this.subscriptions.push(environmentLayerSubscription);
 
-        this.dataSelectionService.setTimeSteps([...generateYearlyTimeSteps(START_YEAR, END_YEAR, this.currentMonth)]);
+        this.dataSelectionService.setTimeSteps(
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/191190545-326cbe63-283f-4026-8478-c02ba5f9c33a.png)

I get lots of errors when starting the application.

Reply:
a260571deb701beab012aa92739ec3fcf7963856

Reply:
![grafik](https://user-images.githubusercontent.com/4947061/191305827-0a6dfa09-d65c-4060-bef5-cf38d1adc881.png)

Unfortunately, I am still getting errors.